src/tv2-common/updatePolicies/shouldRemoveOrphanedPartInstance.ts

```diff
@@ -5,5 +5,5 @@ export function shouldRemoveOrphanedPartInstance(
 	_context: IRundownUserContext,
 	partInstance: BlueprintRemoveOrphanedPartInstance
 ): boolean {
-	return !(partInstance.partInstance.part.metaData as PartMetaData).dirty
+	return !(partInstance.partInstance.part.metaData as PartMetaData | undefined)?.dirty
```

Comment:
We should strive to not let our types be undefined as it forces us to make undefined check later in the code. 
In this case the 'later in the code' part is two spaces right after the declaration of undefined.

But since Sofie is Sofie I assume that this metaData is able to already be undefined when it gets here and that's why you added the undefined type.

Reply:
`metaData` is transparent to Core (it is whatever the blueprints put in there, if they want, because it is optional). So it should be only in the functions exported through the blueprints manifest, that get that data back, where we have to cast to the type we know we put there. Optional values are useful, you just have to be careful when casting 'early in the code'.